server/database/config/db_build.sql

```diff
@@ -0,0 +1,59 @@
+BEGIN; 
+DROP TABLE IF EXISTS student, staff, course, dates, studentcourse CASCADE;
+CREATE TABLE student (
+	id SERIAL  PRIMARY KEY,
+	firstname VARCHAR(20) NOT NULL,
+	lastname VARCHAR(20) NOT NULL,
+    isActive boolean,
+    homePhone VARCHAR(15),
+	mobilePhone VARCHAR(15) NOT NULL,
+    studentlevel INTEGER NOT NULL,
+	password VARCHAR NOT NULL
+);
+
+CREATE TABLE staff (
+	id SERIAL  PRIMARY KEY,
+	firstname VARCHAR(20) NOT NULL,
+	lastname VARCHAR NOT NULL,
+    email varchar NOT NULL,
+	mobilePhone VARCHAR NOT NULL UNIQUE,
+	password VARCHAR NOT NULL
+);
+
+CREATE TABLE course (
+	id SERIAL  PRIMARY KEY,
+	title TEXT NOT NULL,
+	description TEXT NOT NULL,
+    targetLevel INTEGER NOT NULL,
+    numberOfStudent INTEGER NOT NULL,
+    publishDate	VARCHAR NOT NULL
+);
+
+CREATE TABLE dates (
+	id SERIAL PRIMARY KEY,
+    hourfrom INTEGER NOT NULL,
+    hourto INTEGER NOT NULL,
+    courseid INT REFERENCES course(id),
+    dayes VARCHAR NOT NULL
```

Comment:
i think if you make ``` days ``` just a ``` char ``` and if the char equal 1 then that indicate to first option and 2 indicate to second option